content/pandas/concepts/dataframe/terms/resetindex/reset-index.md

```diff
@@ -0,0 +1,168 @@
+---
+Title: '.reset_index()'
+Description: 'This article is about how the .reset_index() is used in pandas dataframes.'
+Subjects: 
+  - 'Data Science'
+  - 'Pandas'
+Tags:
+  - 'Methods'
+  - 'Pandas'
+  - 'Data Science'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/data-science'
+---
+This article is about how the **.reset_index()** is used in pandas dataframes.
+
+
+## Syntax
+```py
+import pandas as pd
+df.read_csv('some_csv.csv', index_col = 'Account Number')
+df.reset_index()
+```
+
+
+### What Is .reset_index() 
+Sometimes there is a need to modify a dataframe object to better meet a certain use case. When these modifications are applied they often leave smaller dataframes with indexes that are messy and not continuous.
+Resetting the index with .reset_index() fixes these issues.
+
+
```

Comment:
```suggestion
```
remove extra line break. There should only be one line break before and after headers.